pydra/tasks/core/sequences.py

```diff
@@ -0,0 +1,115 @@
+import attr
+import typing as ty
+from pydra.engine.specs import BaseSpec, SpecInfo
+from pydra.engine.core import TaskBase
+from pydra.engine.helpers import ensure_list
+
+
+@attr.s(kw_only=True)
+class MergeInputSpec(BaseSpec):
+    axis: ty.Literal["vstack", "hstack"] = attr.ib(
```

Comment:
Fails on Python 3.7, since `Literal` was added in 3.8. Not sure if we have another way of doing this or want to monkey-patch it into 3.7, as in https://github.com/python/typing/blob/b5c0b6d66b3adba4dde3f4f4145ce83a70fd03f7/typing_extensions/src_py3/typing_extensions.py#L543-L676.

Reply:
in some projects we use `sys.version_info` to pull `Literal` in for 3.7

https://github.com/dandi/dandi-cli/blob/master/setup.cfg#L56
https://github.com/dandi/dandi-cli/blob/master/dandi/models.py#L22

Reply:
Ah, `typing_extensions`. I tried it with just `typing` and broke pip.